sr_hand_config/6234/rates/sensor_data_polling.yaml

```diff
@@ -50,5 +50,5 @@ ubi0_sensor_data_update_rate: {
 
 mst_sensor_data_update_rate: {
   TACTILE_SENSOR_TYPE_MST_MAGNETIC_INDUCTION: -1.,
-  TACTILE_SENSOR_TYPE_MST_TEMPERATURE: -1.
-  }
\ No newline at end of file
+  TACTILE_SENSOR_TYPE_MST_TEMPERATURE: 1.
```

Comment:
What is this about? positive or negative integer rates but with decimal points? I guess they're all like this but can someone explain why? seems weird...

Reply:
We should add some comment explaining how this works.

- Positive numbers are frequency of polling in Hz
- -1 means "poll as frequently as possible". This normally means polling at 1Khz if there's only one type of data configured like this. Or polling at 500Hz if two types of data are configured as -1, en so on. That is, unless a positively conigured type of data needs to be polled instead. the "-1" data basically will fill all the gaps when nothing else has to be pollled.
- -2 means that this type of data is only polled once at initialization.

Reply:
Ok, if this comment gets added I'm happy to pass.

Reply:
(even though there are now two reviews. please add comments anyway.

Reply:
Created a separate PR for that:
https://github.com/shadow-robot/sr_hand_config/pull/269